### PR TITLE
Add Example Config

### PR DESCRIPTION
**Pull request by Charles-Schleich**

### CODE REVIEW

**Comment by oteffahi** [comment]
@Charles-Schleich is the additional indentation of the json intended?

File: EXAMPLE_CONFIG.json5 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                // the RocksDB database will be stored in this directory (relative to ${ZENOH_BACKEND_ROCKSDB_ROOT})
+                dir: "example",
+                // create the RocksDB database if not already existing
+                create_db: true
+              }
+            }
+          }
+        },
+      }
+    }
```
Comment (oteffahi) [inline EXAMPLE_CONFIG.json5:28]: Remove unnecessary leading indentation
```suggestion
{
  plugins: {
    // configuration of "storages" plugin:
    storage_manager: {
      volumes: {
        // configuration of a "rocksdb" volume (the "zenoh_backend_rocksdb" backend library will be loaded at startup)
        rocksdb: {}
      },
      storages: {
        // configuration of a "demo" storage using the "rocksdb" volume
        demo: {
          // the key expression this storage will subscribes to
          key_expr: "demo/example/**",
          // this prefix will be stripped from the received key when converting to database key.
          // i.e.: "demo/example/a/b" will be stored as "a/b"
          strip_prefix: "demo/example",
          volume: {
            id: "rocksdb",
            // the RocksDB database will be stored in this directory (relative to ${ZENOH_BACKEND_ROCKSDB_ROOT})
            dir: "example",
            // create the RocksDB database if not already existing
            create_db: true
          }
        }
      }
    },
  }
}
```

**Comment by Charles-Schleich** [comment]
Absolutely not ! :) 
Fixed.

**Review by oteffahi — APPROVED** [review]
LGTM

**Pull request merged**